cockatrice/src/remoteclient.cpp

```diff
@@ -299,6 +343,30 @@ void RemoteClient::doRegisterToServer(const QString &hostname, unsigned int port
     setStatus(StatusRegistering);
 }
 
+void RemoteClient::doForgotPassword(const QString &hostname, unsigned int port, const QString &_userName, const QString &_email)
+{
+	doDisconnectFromServer();
+
+	userName = _userName;
+	email = _email;
```

Comment:
Doesn't look like the email is used... I put a fake email in and it still emailed me at the correct email addr. Function should be it fails to send due to input_email != database_email. Whether we alert the user or not is up to you, but I disagree with having a user input an email addr if we're not using it for validation purposes